codegen/src/api/storage.rs

```diff
@@ -74,12 +74,15 @@ fn generate_storage_entry_fns(
     storage_entry: &StorageEntryMetadata<PortableForm>,
 ) -> (TokenStream2, TokenStream2) {
     let entry_struct_ident = format_ident!("{}", storage_entry.name);
-    let (fields, entry_struct, constructor, key_impl) = match storage_entry.ty {
+    let is_account_wrapper = pallet.name == "System" && storage_entry.name == "Account";
+    let wrapper_struct_ident = format_ident!("{}DefaultData", storage_entry.name);
```

Comment:
nitpicking: I'm not super fond of the name `AccountDefaultData`; I'd lean towards something like `AccountOwned` I guess.. but I'm not too fussed :D

Reply:
`AccountOwned` sounds better :D Thanks